charts/fluid/fluid/templates/crd-upgrade.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: fluid-crds-upgrade
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "-5"
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+data:
+  upgrade-crds.sh: |
+    #!/bin/bash
+    set -e
+
+    for crdfile in $(find charts/fluid/fluid/crds/*.yaml);
+    do
+      crdshort=${crdfile#*_}
+      if [[ $(kubectl get --ignore-not-found -f $crdfile | wc -l) -gt 0 ]]; then
+        echo "$crdshort founded, replacing its crd..."
+        kubectl replace -f $crdfile
+      else
+        echo "$crdshort not founded, applying its crd..."
+        kubectl apply -f $crdfile
+      fi
+    done
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: fluid-crds-upgrade
```

Comment:
I think it's safer to make the name with random suffix.

Reply:
Done